neuromation/api/config.py

```diff
@@ -133,6 +133,7 @@ class _Config:
     version: str
     cluster_name: str
     clusters: Mapping[str, Cluster]
+    default_run_time_limit: timedelta
```

Comment:
Why `default_run_time_limit` is a part of config?
It is explicitly specified by `jobs.run()` argument.
If you want to provide user-default value for the limit -- see `neuromation/cli/jobs:calc_filters` for how to do it properly.